src/stability_sdk/client.py

```diff
@@ -240,12 +240,19 @@ def update(self, title:str=None, file_id:str=None, file_uri:str=None):
             use=project.PROJECT_ASSET_USE_PROJECT,
         ) if file_id and file_uri else None
         
-        self._project = self._api._proj_stub.Update(project.UpdateProjectRequest(
+        self._api._proj_stub.Update(project.UpdateProjectRequest(
             id=self.id, 
             title=title,
             file=file
         ))
 
+        if title:
```

Comment:
might be a good idea to check that `self._api._proj_stub.Update` returned a "success" response before propagating the new fields to the local `_project` attribute just to make sure everything stays in synch. I'm not sure how big of an issue it would be if the `Update` api call fails and the local project gets its values updated anyway. Probably pretty low risk since `self.id` stays fixed. Just something to think about.